test/verify/check-shell-active-pages

```diff
@@ -19,28 +19,23 @@
 
 import parent
 from testlib import *
-from machine_core.constants import TEST_OS_DEFAULT
 
 
 @skipImage("Do not test BaseOS packages", "rhel-8-3-distropkg", "rhel-8-4-distropkg")
+@nondestructive
 class TestActivePages(MachineCase):
 
     def testBasic(self):
         b = self.browser
         m = self.machine
 
-        self.login_and_go("/system")
+        # disable preloads; they are different between OSes, make the counting hard, and happen asynchronously
+        # disable_preload(), sed_file() etc. need writable file system
+        for p in ["packagekit", "playground", "systemd"]:
+            self.restore_dir("/usr/share/cockpit/" + p)
```

Comment:
I was just thinking, this should actually go into the implementation of `disable_preload()`. We don't use it anywhere on OSTree until now. I'll wait for a round of test results and then update that.

Reply:
Previous round's results all succeeded everywhere, pushed this cleanup.